data-prepper-plugins/service-map-stateful/src/main/java/com/amazon/dataprepper/plugins/prepper/ServiceMapStatefulPrepper.java

```diff
@@ -253,6 +258,18 @@ private static File createPath(File path) {
         return serviceDependencyRecords;
     }
 
+    public static void main(String[] args) {
+        final ServiceMapRelationship destinationRelationship =
+                ServiceMapRelationship.newDestinationRelationship("serviceA",
+                        "kindA", "serviceB", "resourceA", "traceGroupA");
+        final ServiceMapRelationship targetRelationship = ServiceMapRelationship.newTargetRelationship("serviceB",
+                "kindB", "serviceB", "traceGroupB", "traceGroupB");
+        final Event targetRelationshipEvent = JacksonEvent.builder().withEventType("event").withData(targetRelationship).build();
+        final Event destinationRelationshipEvent = JacksonEvent.builder().withEventType("event").withData(destinationRelationship).build();
+        System.out.println(targetRelationshipEvent.toJsonString());
+        System.out.println(destinationRelationshipEvent.toJsonString());
+    }
```

Comment:
Is this test code? If so, please remove. If not, can you explain why ServiceMapStatefulPrepper needs a main method?

Reply:
My bad. Should be removed